CHANGELOG.md

```diff
@@ -158,6 +185,13 @@ All notable, unreleased changes to this project will be documented in this file.
   `CheckoutAddPromoCode`, `CheckoutPaymentCreate` will raise a ValidationError when product in the checkout is
   unavailable - #8978 by @IKarbowiak
 - Add `withChoices` flag for Attribute type - #7733 by @dexon44
+`CheckoutAddPromoCode`, `CheckoutPaymentCreate` will raise a ValidationError when product in the checkout is
```

Comment:
It looks like something that is not the scope of this PR.

Reply:
Resolved in recent commit.